common/components/controllers/LogInController.jsx

```diff
@@ -71,10 +72,28 @@ class LogInController extends React.Component<Props, State> {
                 Sign In
               </button>
             </div>
+            <div className="LogInController-socialSection">
+              <div className="centered">
```

Comment:
I'd change this to `text-center` instead. ref: https://getbootstrap.com/docs/4.0a/utilities/text/
We could probably just apply it to the <p> and remove the nested div here as well.